bio2zarr/tskit.py

```diff
@@ -12,31 +12,40 @@ class TskitFormat(vcz.Source):
     @core.requires_optional_dependency("tskit", "tskit")
     def __init__(
         self,
-        ts_path,
-        individuals_nodes=None,
-        sample_ids=None,
+        ts,
+        *,
+        model_mapping=None,
         contig_id=None,
         isolated_as_missing=False,
     ):
         import tskit
 
-        self._path = ts_path
-        self.ts = tskit.load(ts_path)
+        self._path = None  # Not sure what we're using this for?
```

Comment:
The base class defines `Source.path` which is used when we process out-of-core to put the path in metadata. This isn't used for tskit (at least for now) so I guess `TskitFormat.path` could just raise `NotImplemented` and the path not be stored.